balancer/xds/edsbalancer/util_test.go

```diff
@@ -20,8 +20,49 @@ package edsbalancer
 
 import (
 	"testing"
+
+	"google.golang.org/grpc/balancer/internal/wrr"
 )
 
+// testWRR is a deterministic WRR implementation.
```

Comment:
Can you explain in the comment why we need the deterministic WRR implementation for testing?

Reply:
Done